### DISCUSSION TITLE
get notification on download finished

### DISCUSSION THREAD

**Discussion by rubenixnagios** [Forceu/Gokapi#174] [Ideas]
First of all thanks for such a nice piece of work @Forceu 
for my use case would love having a notification system (maybe api call) once a file gets downloaded, error, gets deleted etc etc
Im using [ntfy](https://github.com/binwiederhier/ntfy) maybe it can be integrated with gokapi somehow; its just an idea.
Im not a coder but can help testing if someone interested on deploying this!!
again: thanks so much!

**Comment by Forceu** [comment]
Tracked in #175

That would be possible, the easiest implementation would be to add it to the SSE stream. Please note however, that it is not possible to know when a download is finished, if the data is stored in the cloud without proxying it. What would be your use case for this feature?

**Reply by rubenixnagios** [reply]
in my case data is stored locally not in cloud.
maybe get notification on download start
will keep an eye on future releases. thanks!
